notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    BATCH_SIZE = 100
</pre><p>Why is 100 used? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>This is what is used in the paper.</p>

Reply:
closing conversation, resolved